static/app/components/events/interfaces/crashContent/exception/utils.tsx

```diff
@@ -33,6 +42,90 @@ interface RenderLinksInTextProps {
   exceptionText: string;
 }
 
+export const getAppleCrashReportEndpoint = (
+  organization: Organization,
+  type: 'original' | 'minified',
+  projectSlug: string,
+  eventId: string
+) => {
+  const minified = type === 'minified';
+  return `/projects/${organization.slug}/${projectSlug}/events/${eventId}/apple-crash-report?minified=${minified}`;
+};
+
+export const getContent = (
+  isNative: boolean,
+  exc: any,
+  type: 'original' | 'minified',
+  projectSlug: string,
+  eventId: string,
+  api: Client,
+  platform?: PlatformKey,
+  loading?: boolean,
+  error?: boolean,
+  crashReport?: string,
+  organization?: Organization
+) => {
```

Comment:
there's probably something else we can do here. looks like we wanted both a component and to use the output